modules/codemod/lib/v5/renameLabsTokenImports.ts

```diff
@@ -0,0 +1,44 @@
+import {API, FileInfo} from 'jscodeshift';
+
+export default function transformer(file: FileInfo, api: API) {
+  const j = api.jscodeshift;
+
+  const root = j(file.source);
+
+  root
+    .find(j.ImportDeclaration, {source: {value: '@workday/canvas-kit-labs-react/tokens'}})
+    .forEach(nodePath => {
+      let shouldRemoveNodePath = false;
+      nodePath.value.specifiers?.forEach(specifier => {
+        // Only iterate over named import statements
+        if (specifier.type === 'ImportSpecifier') {
+          // Rename type imports from "labs/tokens" to "/tokens"
+          if (specifier.imported.name === 'type') {
+            nodePath.insertBefore(
```

Comment:
I think you mentioned this in standup, but this could lead to duplicate import declarations

Reply:
Yeah, thanks. I have a test (currently commented out) to catch duplicate imports. I still need to figure out how to merge imports.